orchagent/portsorch.cpp

```diff
@@ -1768,19 +1771,19 @@ void PortsOrch::initializeQueues(Port &port)
                 sai_serialize_object_id(port.m_port_id));
         queuePortVector.push_back(queuePortTuple);
 
-        FieldValueTuple queueIndexTuple(
-                sai_serialize_object_id(port.m_queue_ids[queueIndex]),
-                to_string(queueIndex));
-        queueIndexVector.push_back(queueIndexTuple);
-
-
         string queueType;
-        if (getQueueType(port.m_queue_ids[queueIndex], queueType))
+        uint8_t queueRealIndex = 0;
+        if (getQueueTypeAndIndex(port.m_queue_ids[queueIndex], queueType, queueRealIndex))
```

Comment:
>getQueueTypeAndIndex [](start = 12, length = 20)

handle return code?

Reply:
Thanks for review, will have error syslog in "getQueueTypeAndIndex" if failed, so here use the same logic as code in left before. i think "getQueueTypeAndIndex" fail is not that critical, the initialize could continue, we can check syslog about this error
what do you suggest?